### DISCUSSION TITLE
API PHP Files

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#5599] [Q&A]
Hi i've a question.

Can you help me to create a PHP Script to get Data from the API.

Sample https://demo.azuracast.com/api/station/1/files

Get "Text" from this json data to an array

**Comment by KazeNoTachi** [comment]
Assuming you already can pull the response as a string from the api you can use json_decode() to convert it to an array:

`json_decode($responseString, true);`

Not sure how it will deal with nested json, as I am not super versed in php. 

Source: https://stackoverflow.com/questions/40714593/how-can-i-convert-my-api-response-into-php-array

**Comment by Sakretsos** [comment]
Hello @LauraS-Development,
Try to use the bellow piece of code!

```
<?php
$url = 'https://demo.azuracast.com/api/station/1/files';
$api_key = 'API_KEY';

$curl = curl_init();
curl_setopt_array($curl, array(
  CURLOPT_URL => $url,
  CURLOPT_RETURNTRANSFER => true,
  CURLOPT_ENCODING => '',
  CURLOPT_MAXREDIRS => 10,
  CURLOPT_TIMEOUT => 0,
  CURLOPT_FOLLOWLOCATION => true,
  CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
  CURLOPT_CUSTOMREQUEST => 'GET',
  CURLOPT_HTTPHEADER => array(
    'Authorization: Bearer ' . $api_key
  ),
));
$response = curl_exec($curl);
curl_close($curl);

$output = json_decode($response, true);

var_dump($output);
?>
```
